bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/i18n/pulseaudio.properties

```diff
@@ -41,37 +41,37 @@ thing-type.config.pulseaudio.combinedSink.name.label = Name
 thing-type.config.pulseaudio.combinedSink.name.description = The name of the combined sink.
 thing-type.config.pulseaudio.sink.activateSimpleProtocolSink.label = Create an Audio Sink with simple-protocol-tcp
 thing-type.config.pulseaudio.sink.activateSimpleProtocolSink.description = Activation of a corresponding sink in OpenHAB (module-simple-protocol-tcp must be available on the pulseaudio server)
+thing-type.config.pulseaudio.sink.additionalFilters.label = Additional Filters
+thing-type.config.pulseaudio.sink.additionalFilters.description = Additional filters to select the proper device on the pulseaudio server, in case of ambiguity. To be selected, the device should have at least a property value matching this regular expression. You can use multiple regular expressions (separator ###).
 thing-type.config.pulseaudio.sink.name.label = Name
 thing-type.config.pulseaudio.sink.name.description = The name of one specific device. You can also use the description.
-thing-type.config.pulseaudio.sink.additionalFilters.label = Additional Filters
-thing-type.config.pulseaudio.sink.additionalFilters.description = Additional filters to select the proper device on the pulseaudio server, in case of ambiguity. To be selected, the device should have at least a property value matching this regular expression. You can use multiple regular expressions (separator is ###).
```

Comment:
Maybe the old description with "separator is ###)." was better ?
If we agree, I would suggest to update XML description to align the description with the previous default translation.

Same for few others.